html_xblock/static/css/html.css

```diff
@@ -1,44 +1,62 @@
-/* .modal-type-html5 [class*="view-"] .modal-lg.modal-editor .modal-content {
-    height: 500px;
-} */
+/* fall back to default browser styles, if border attribute is not set; the inline styles (like border style and color) would overwrite it */
+.studio-xblock-wrapper [data-block-type="html5"] table:not([border]) {
+  border: unset;
+}
+
+/* default styles for any table that had border attribute set; the inline styles (like border style and color) would overwrite it */
+.studio-xblock-wrapper [data-block-type="html5"] table[border] {
+  border: 1px solid black;
+}
+
+/* change the border width, depending on the explicit value of the border attribute */
+.studio-xblock-wrapper [data-block-type="html5"] table[border="1"] {
+  border-width: 1px;
+}
+
+.studio-xblock-wrapper [data-block-type="html5"] table[border="2"] {
+  border-width: 2px;
+}
+
+.studio-xblock-wrapper [data-block-type="html5"] table[border="3"] {
+  border-width: 3px;
+}
 
-.html-editor,
-.wrapper-comp-settings.is-active.editor-with-buttons {
-    height: 320px;
+.studio-xblock-wrapper [data-block-type="html5"] table[border="4"] {
+  border-width: 4px;
 }
 
-[class*="view-"] .modal-window .CodeMirror,
-[class*="view-"] .modal-window .editor-with-buttons.wrapper-comp-settings .list-input.settings-list {
-    height: 322px;
+.studio-xblock-wrapper [data-block-type="html5"] table[border="5"] {
+  border-width: 5px;
 }
 
-/* Style the tab */
-.tab {
-    overflow: hidden;
-    border: 1px solid #ccc;
-    background-color: #f1f1f1;
+.studio-xblock-wrapper [data-block-type="html5"] table[border="6"] {
+  border-width: 6px;
 }
 
-/* Style the buttons inside the tab */
-.tab button {
-    background-color: inherit;
-    float: left;
-    border: none;
-    outline: none;
-    cursor: pointer;
-    padding: 14px 16px;
-    transition: 0.3s;
-    font-size: 17px;
+.studio-xblock-wrapper [data-block-type="html5"] table[border="7"] {
+  border-width: 7px;
 }
 
-.tab button:hover {
-    background-color: #ddd;
+.studio-xblock-wrapper [data-block-type="html5"] table[border="8"] {
+  border-width: 8px;
 }
 
-.tab button.active {
-    background-color: #ccc;
+.studio-xblock-wrapper [data-block-type="html5"] table[border="9"] {
+  border-width: 9px;
 }
 
-.tabcontent {
-    display: none;
+.studio-xblock-wrapper [data-block-type="html5"] table[border="10"] {
+  border-width: 10px;
+}
+
+/* etc until the value we think is reasonable */
+
+.studio-xblock-wrapper [data-block-type="html5"] table[border] > * > tr > td,
+.studio-xblock-wrapper [data-block-type="html5"] table[border] > * > tr > th,
+.studio-xblock-wrapper [data-block-type="html5"] table[border] > * > td,
+.studio-xblock-wrapper [data-block-type="html5"] table[border] > * > th,
+.studio-xblock-wrapper [data-block-type="html5"] table[border] > td,
+.studio-xblock-wrapper [data-block-type="html5"] table[border] > th {
+  border-width: thin;
+  border-style: inset;
 }
```

Comment:
Yeah, I definitely missed that case. It's even visible on the screenshots. I guess I've been staring at it for too long...

Reply:
I've fixed it by using `border: unset;`, since without global styles the default border width for the rows would not be `border-width: thin;`. I've also updated the screenshot in the PR description.